docs/03-developer-tools/00-sugar/03-guides/04-sugar-for-cmv3.md

```diff
@@ -0,0 +1,564 @@
+# Sugar for Candy Machine V3
+
+:::caution
+Sugar support for Candy Machine V3 is experimental.
+:::
+
+The Candy Machine V3 is the latest iteration of the Metaplex Protocol's Candy Machine minting and distribution program. While the deploy of a Candy Machine V3 works largely similar to the previous V2 version, Sugar offers a set of new command to configure and manage a Candy Guard.
+
+The Candy Machine V3 has a modular architecture, where the mint logic is separated from the mint access control &mdash; check the [overview](../../../01-programs/02-candy-machine/00-overview.md) to read more about Candy Machine V3 and Candy Guard.
+
+This guide will walk throught the changes in the configuration for a Candy Machine V3 and the commands used to set up and manage a Candy Guard. It assumes that you are already familiar with how Sugar works and have deployed a Candy Machine &mdash; if that is not the case, we recommend to first complete the [My First Candy Machine](../02-tutorials/00-my-first-candy-machine.md) tutorial.
+
+## Configuration
+
+Since the configuration of the mint process has moved to **guards**, Sugar's config file has changed. A basic configuration without any guards is shown bellow:
+
+```json
+{
+  "number": 10,
+  "symbol": "TEST",
+  "sellerFeeBasisPoints": 500,
+  "isMutable": true,
+  "isSequential": false,
+  "creators": [
+    {
+      "address": "PanbgtcTiZ2PveV96t2FHSffiLHXXjMuhvoabUUKKm8",
+      "share": 100
+    }
+  ],
+  "uploadMethod": "bundlr",
+  "awsConfig": null,
+  "nftStorageAuthToken": null,
+  "shdwStorageAccount": null,
+  "pinataConfig": null,
+  "hiddenSettings": null,
+  "guards": null
+}
+```
+ There are two new elements in the configuration file:
+
+ - `isSequential`: indicates to whether a sequential index generation should be used during mint or not (recommended to set this value to `false`);
+ - `guards`: indicates the configuration for the Candy Guard. If this value is set to `null`, a Candy Guard will not be used and minted will only be possible using the `mint_authority` of the Candy Machine.
+
+ :::info
+ You can use `sugar create-config` to create a basic configuration file. The Candy Guard configuration needs to be added manually.
```

Comment:
How?

Reply:
I tweak a bit the text, but I am not sure if I address the `how?` completely.

Reply:
So sorry! How does one create a guard config manually? Is that from the below text? If so, a phrase like "further explained below" or something way better than that.

Reply:
Added the text "further explained below" at the end of the sentence.